package-lock.json

```diff
@@ -0,0 +1,1742 @@
+{
```

Comment:
I haven't really decided about locking strategy so let's continue to procrastinate on this decision. I don't think we need to introduce this file right now.